src/index-sdk/rewards.ts

```diff
@@ -0,0 +1,74 @@
+import Web3 from 'web3'
+import { provider } from 'web3-core'
+import { AbiItem } from 'web3-utils'
+import MerkleABI from './abi/MerkleDistributor.json'
+import merkleData from './novemberRewardsMerkle.json'
+
+export const getMerkleContract = (provider: provider) => {
+  const web3 = new Web3(provider)
+  return new web3.eth.Contract(
+    (MerkleABI as unknown) as AbiItem,
+    process.env.REACT_APP_NOVEMBER_REWARDS_ADDRESS
```

Comment:
typically we import/export all eth contract addresses in `ethContractAddresses.ts`